app/services/order_cycle_form.rb

```diff
@@ -0,0 +1,13 @@
+class OrderCycleForm
+  attr_accessor :order_cycle, :params
+
+  def initialize(order_cycle, params)
+    @order_cycle = order_cycle
+    @params = params
+  end
+
+  def save
+    order_cycle.assign_attributes(params[:order_cycle])
+    order_cycle.save
+  end
+end
```

Comment:
do we really need this? I don't see the value this brings now. It's possibly the only controller action that's still simple, except for its success logic, which could deserve being extracted.